core/sup_tree_core/executor_pool/memcache_manager.ex

```diff
@@ -0,0 +1,103 @@
+# Copyright(c) 2015-2018 ACCESS CO., LTD. All rights reserved.
+
+use Croma
+
+defmodule AntikytheraCore.ExecutorPool.MemcacheManager do
+  @moduledoc """
+  A `GenServer` to manage memcache for each executor pool.
+  """
+
+  @timeout         300_000
+  @max_records_num 10
+  @max_record_size 65536
+
+  use GenServer
+  alias AntikytheraCore.Ets.Memcache
+  alias AntikytheraCore.ExecutorPool.RegisteredName, as: RegName
+  alias Antikythera.ExecutorPool.Id, as: EPoolId
+
+  def start_link(name, epool_id) do
+    GenServer.start_link(__MODULE__, epool_id, [name: name])
+  end
+
+  @impl true
+  def init(epool_id) do
+    {:ok, %{epool_id: epool_id, records: %{}, expire_at_set: :gb_sets.new()}, @timeout}
+  end
+
+  @impl true
+  def handle_call({:write, key, value, lifetime, ratio}, _from, state) do
+    evicted_state = evict_expired_records(state, System.monotonic_time(:milliseconds))
+    if :erts_debug.flat_size(value) > @max_record_size do
```

Comment:
Isn't it possible to off-load this value size checks to caller processes, so that this `GenServer`'s task is minimized?
I think we probably won't have any needs for making "max value size" customizable for each epool.
(If we would then the value size checks should be done in this `GenServer`.)
On the other hand we probably will make "max number of cached objects" customizable, but this is out of scope of this PR.

Reply:
The limit was removed.